### DISCUSSION TITLE
Blog 3

### DISCUSSION THREAD

**Discussion by Anastylos** [Anastylos/Jump-and-Run#48] [Blog]
Link zu unserem SRS:https://github.com/Anastylos/Jump-and-Run/blob/main/Dokumente/SRS_Gamedesign.md

Diese Woche haben wir Aktivitätsdiagramme und Sequenzdiagramme erstellt:

![2024-10-29_Aktivitätsdiagramm_Hauptmenue](https://github.com/user-attachments/assets/a9df319c-0488-4cd0-84b5-4ea76d93be06)
![2024-10-29_Aktivitätsdiagramm_Totem](https://github.com/user-attachments/assets/908d234f-9745-4b14-bcbe-f9ac29396fcf)
![2024-10-29_Sequenzdiagramm_SpielerBewegung](https://github.co
![204-10-29_Aktivitätsdiagramm_Seilzug](https://github.com/user-attachments/assets/c45613fa-11b2-4cb6-8060-bea8515676fe)
m/user-attachments/assets/df72f56e-4a9b-47fe-8519-1f62a846f8f8)
![2024-10-29-Sequenzdiagramm_Speichermanagement](https://github.com/user-attachments/assets/a5737422-50b9-44ec-819f-61539ebb9a5a)

**Comment by OmarMoujahed** [comment]
Wir schätzen es sehr, dass ihr die verschiedenen Themenbereiche klar strukturiert habt. Besonders hervorzuheben ist, dass durch die Aktivitätsdiagramme der Ablauf des Spieles und die verschiedenen Funktionen sehr leicht zu verstehen sind. Die Sequenzdiagramme tragen ebenfalls dazu bei, die Funktionalität des geplanten Systems besser zu erfassen.

Das Sequenzdiagramm ist klar strukturiert und zeigt den Ablauf zwischen Hauptmenü, Spiel und Savefile nachvollziehbar auf. Die Nutzung des optionalen Blocks (opt) für den Restart ist sinnvoll. Verbesserungen könnten durch präzisere Beschriftungen und eine klare Trennung zwischen aktiven und passiven Objekten erzielt werden. Insgesamt eine gelungene Darstellung mit kleineren Optimierungsmöglichkeiten.

Wir freuen uns schon sehr auf die Entwicklungen in der nächsten Woche!

Viele Grüße,
Project HÆLTH 🌱
